worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/PartitionDataWriter.java

```diff
@@ -416,6 +419,8 @@ public boolean isClosed() {
     return closed;
   }
 
+  public void setWriterClosed() {}
```

Comment:
Is it necessary to introduce `setWriterClosed` method? IMO, no.

Reply:
Good catch, this method can be removed indeed.